README.md

```diff
@@ -38,6 +38,7 @@ $ ninja -C build install # or `meson install -C build`
 $ rizin
 Usage: rizin [-ACdfLMnNqStuvwzX] [-P patch] [-p prj] [-a arch] [-b bits] [-i file]
              [-s addr] [-B baddr] [-m maddr] [-c cmd] [-e k=v] file|pid|-|--|=
```

Comment:
@ret2libc I personally don't like showing examples with output that can change. When a command line flag will be removed or added, nobody will remember to update this and it will eventually stay outdated and msileading. What do you think?

Reply:
True... but I want to show a simple thing to check if rizin is correctly installed. Do you have other suggestions?

Reply:
`rizin --version` ?

Reply:
At the moment that's even worse, because we have to change how that looks because of the change of the version and because of a bug in how gittap,tip,top,tup are calculated.

Reply:
I would say to go back to this problem in a while and leave it as-is for now, though it's a valid point.

Reply:
makes sense.

Reply:
version is also. something that changes.
Maybe "$ rizin /bin/ls" or "$ rizin demo.exe"?

Reply:
What about `rizin --`?